clients/admin-ui/cypress/e2e/nav-bar.cy.ts

```diff
@@ -8,7 +8,7 @@ describe("Nav Bar", () => {
   it("renders all navigation groups with links inside", () => {
     cy.visit("/");
 
-    cy.get("nav button").should("have.length", 7);
+    cy.get("[id^='accordion-button']").should("have.length", 4);
```

Comment:
After updating uses of `NextLink`s, some of the elements are an `<a>` now depending on what they do. So this update is for testing for the actual group headings instead, as the test name suggests.